keras_cv/bounding_box/iou.py

```diff
@@ -14,27 +14,41 @@
 """Contains functions to compute ious of bounding boxes."""
 import tensorflow as tf
 
+from keras_cv import bounding_box
 
-def compute_ious_for_image(boxes1, boxes2):
-    """computes a lookup table vector containing the ious for a given set boxes.
 
-    The lookup vector is to be indexed by [`boxes1_index`,`boxes2_index`].
+def compute_ious_for_image(boxes1, boxes2, bounding_box_format):
```

Comment:
let's rename this to `compute_iou()`

let's also introduce batched support to this now that this is a utility.  Feel free to do batched support using map_fn, performance is not a huge concern for batched ATM, mainly concerned about API consistency.